filter/unquote_resolvers.py

```diff
@@ -0,0 +1,79 @@
+from typing import Any, Dict, List, Union
+
+import yaml
+from yaml.nodes import ScalarNode
+
+try:
+    from yaml import CSafeDumper as SafeDumper
```

Comment:
When we have to do stuff like this, it's useful to add a comment as to why, since it's otherwise not apparent.

Reply:
Isn't it fairly obvious? If the C SafeDumper is available, use that because it's faster; otherwise use SafeDumper.